src/main/java/com/powsybl/openloadflow/network/impl/HvdcConverterStations.java

```diff
@@ -45,27 +44,31 @@ public static boolean isVsc(Identifiable<?> identifiable) {
                 && ((HvdcConverterStation<?>) identifiable).getHvdcType() == HvdcConverterStation.HvdcType.VSC;
     }
 
-    public static boolean isHvdcDanglingInIidm(HvdcConverterStation<?> station, LfNetworkParameters parameters) {
+    public static boolean isHvdcDanglingInIidm(HvdcConverterStation<?> station) {
 
-        if (isIsolated(station.getTerminal(), parameters)) {
+        if (isIsolated(station.getTerminal())) {
             return true;
         } else {
             return station.getOtherConverterStation().map(otherConverterStation -> {
                 Terminal otherTerminal = otherConverterStation.getTerminal();
-                return isIsolated(otherTerminal, parameters);
+                return isIsolated(otherTerminal);
             }).orElse(true); // it means there is no HVDC line connected to station
         }
     }
 
-    private static boolean isIsolated(Terminal terminal, LfNetworkParameters parameters) {
-        Bus bus = parameters.isBreakers() ? terminal.getBusBreakerView().getBus() : terminal.getBusView().getBus();
+    private static boolean isIsolated(Terminal terminal) {
+        Bus bus = terminal.getBusView().getBus();
```

Comment:
I would rather agree on this change, but has it been motivated by something failing?

Reply:
No. Just the fact that when I wrote it first I didn't fully understand the difference betwwen BusView and BusBreakerVew

Reply:
Indead it seems to be only called at Lf Network loading so only one time.

Reply:
So maybe we should revert this change and think about it in another one.

Reply:
Oh la la... So it can breaks something in a topology I have seen where HVDC disconnection matters. I'll show you next time we meet.

Reply:
So in that case, you just need to add a unit test that shows the issue.

Reply:
Done. That was a bit trickier that I thought - and the bug is more seldom that I though because:
   - the breaker mode is activated in very rare occasions
   - the criteria is only used for an HVDC part that is not in the LfNetwork
But the bug existed.